test/integration/agent/openflow_test.go

```diff
@@ -1408,7 +1408,7 @@ func TestSNATFlows(t *testing.T) {
 	require.Nil(t, err, fmt.Sprintf("Failed to prepare OVS bridge %s", br))
 
 	config := prepareConfiguration()
-	_, err = c.Initialize(roundInfo, config.nodeConfig, &config1.NetworkConfig{TrafficEncapMode: config1.TrafficEncapModeEncap})
+	_, err = c.Initialize(roundInfo, config.nodeConfig, &config1.NetworkConfig{TrafficEncapMode: config1.TrafficEncapModeEncap}, &config1.EgressConfig{ExceptCIDRs: []net.IPNet{}})
```

Comment:
could we add values to ExceptCIDRs and validate the desired OpenFlow rules are present to cover the scenario?

Reply:
OK, I will add the scenario case into testExternalFlows